qiskit/tools/visualization/_qcstyle.py

```diff
@@ -99,4 +99,4 @@ def set_style(self, dic):
         self.dpi = dic.get('dpi', self.dpi)
         self.margin = dic.get('margin', self.margin)
         self.cline = dic.get('creglinestyle', self.cline)
-        self.reverse = dic.get('reversebits', self.reverse)
+        self.reverse_bits = dic.get('reversebits', self.reverse_bits)
```

Comment:
This can actually be removed, it was missed as part of https://github.com/Qiskit/qiskit-terra/pull/1567 which removed support for using `reverse_bits` and `plot_barriers` from the style dict.

Reply:
I have removed it